doc/sphinx/language/core/modules.rst

```diff
@@ -511,6 +511,249 @@ transparent constraint
 Notice that ``M`` is a correct body for the component ``M2`` since its ``T``
 component is ``nat`` as specified for ``M1.T``.
 
+.. _qualified-names:
```

Comment:
If you want to move a large block of text, please do that by itself in a separate commit or separate PR with a comment like "Move qualified names section" so it's easy to see the changes.  Also would help to eventually squash some of your commits so the changes are easy to pick out.

Reply:
Alright, I can do that with a little bit of git struggle next weak.

Reply:
Use `git rebase -i HEAD~999`, substituting something sensible for the `999` to move the text before your other changes.  Repeat moving the text and commit that before `git rebase continue`.  (Glossing over some details.  I'd be happy to be more explicit if you like.)

Reply:
A good try, but the "Move Typing Modules" commit also includes the added table.  If you want to give it another try or would like help, let me know.  It's not the end of the world if it remains as you have it now.

Reply:
Don't forget about this one (make "[Move "Typing modules" to the end](https://github.com/coq/coq/pull/19323/commits/9710d79eefed34a9c0465c1fc77fe1c7b949b409)" only moving text, not adding the table).